### PR TITLE
Topology coming soon blog

### LINKED ISSUE #15: ✍🏼 Blog: Introducing our new Topology plugin
## Goal

Post a blog on janus-idp.io introducing our new Topology plugin!

### PR DESCRIPTION
**Pull request by serenamarie125**
## What does this PR do / why we need it
New Topology coming soon blog

Fixes #15

### CODE REVIEW

File: apps/landing-page/pages/blog/_meta.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
       "typesetting": "article"
     }
   },
+  "topology-plugin-coming-soon": "Topology plugin coming soon to a Backstage near you!",
```
Comment (schultzp2020) [inline apps/landing-page/pages/blog/_meta.json:12]:
```suggestion
  "topology-plugin-coming-soon-to-a-backstage-near-you": "Topology plugin coming soon to a Backstage near you!",
```

You will also need to rename the mdx file and image directory name.

File: apps/landing-page/pages/blog/topology-plugin-coming-soon.mdx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import { Authors } from 'ui/components';
+
+<Authors authors={['_div_gupta']} />
+
+# Topology plugin coming soon to a Backstage near you!
+
+The Janus community is thrilled to share details about our Backstage Topology Plugin. This powerful new tool simplifies the process of visualizing k8s workloads of your Backstage services. With this plugin, developers can get a clear and concise overview of their application's structure and workload status. This eliminates the stress and cognitive overload that often comes with working with Kubernetes.
+
+![Topology plugin coming soon!](/images/blog/topology-plugin-coming-soon/topologyplugin.png)
```
Comment (schultzp2020) [inline apps/landing-page/pages/blog/topology-plugin-coming-soon.mdx:15]:
```suggestion
![Topology plugin coming soon!](/images/blog/topology-plugin-coming-soon/topology-plugin.png)
```

You will also need to rename the image file. Its generally best practice to name files in file case.

**Review by schultzp2020 — CHANGES_REQUESTED** [review]

**Review by schultzp2020 — CHANGES_REQUESTED** [review]
[topologyplugin.png](https://github.com/janus-idp/janus-idp.github.io/pull/38/files?short_path=7a4ce32#diff-7a4ce32f92443e5f17d75d4c1e214d438b8cce8cec34a4855acc976b1bdadb90) needs to be renamed to `topology-plugin.png`

**Comment by serenamarie125** [comment]
Preparing another PR

**Pull request closed without merging**